bug: quickgui fails to build on NixOS

**I confirm this bug has not already been reported**
- [ x] I have searched the issues and this bug has not been reported previously

**Description**
When building/installing the package on NixOS, errors occur that halt the build process.

**To Reproduce**
add quickgui to `environment.systemPackages`
run `sudo nixos-rebuild switch`

**Expected behavior**
Successful Build

**Desktop:**
 - Quickgui Version: 1.2.10
 - OS: NixOS
 - Version: unstable

**Additional context**
Errors from build:
```
quickgui> [   +8 ms] FAILED: /build/source/linux/flutter/ephemeral/libflutter_linux_gtk.so /build/source/linux/flutter/ephemeral/flutter_linux/fl_basic_message_channel.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_binary_codec.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_binary_messenger.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_dart_project.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_engine.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_json_message_codec.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_json_method_codec.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_message_codec.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_method_call.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_method_channel.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_method_codec.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_method_response.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_plugin_registrar.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_plugin_registry.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_standard_message_codec.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_standard_method_codec.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_string_codec.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_value.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_view.h /build/source/linux/flutter/ephemeral/flutter_linux/flutter_linux.h flutter/_phony_ /build/source/build/linux/x64/release/flutter/_phony_ 

...

quickgui> [        ] [  +10 ms] Failed obtaining PRETTY_NAME for Linux: PathNotFoundException: Cannot open file, path = '/usr/lib/os-release' (OS Error: No such file or directory, errno = 2)

...

quickgui> [        ] [+4813 ms] /nix/store/bgvqb8pa4vspr3abxxyww7fdxqdqwmjr-flutter-sdk-flutter/lib/src/material/color_scheme.dart:1915:39: Error: Method not found: 'SchemeRainbow'.
quickgui> [        ] [   +4 ms]       DynamicSchemeVariant.rainbow => SchemeRainbow(sourceColorHct: sourceColor, isDark: isDark, contrastLevel: contrastLevel),
quickgui> [        ] [        ]                                       ^^^^^^^^^^^^^
quickgui> [        ] [        ] /nix/store/bgvqb8pa4vspr3abxxyww7fdxqdqwmjr-flutter-sdk-flutter/lib/src/material/color_scheme.dart:1916:42: Error: Method not found: 'SchemeFruitSalad'.
quickgui> [        ] [        ]       DynamicSchemeVariant.fruitSalad => SchemeFruitSalad(sourceColorHct: sourceColor, isDark: isDark, contrastLevel: contrastLevel),
quickgui> [        ] [        ]                                          ^^^^^^^^^^^^^^^^

```
Full logs from build:
```
quickgui> building '/nix/store/zyfgvki44wpdagcdmwc429ny2qn2h1p3-quickgui-1.2.10.drv'
quickgui> Running phase: unpackPhase
quickgui> unpacking source archive /nix/store/72igdr4x2jvjsv9jrhq1p42837svircm-source
quickgui> source root is source
quickgui> Running phase: patchPhase
quickgui> Running phase: updateAutotoolsGnuConfigScriptsPhase
quickgui> Running phase: configurePhase
quickgui> no configure script, doing nothing
quickgui> Executing dartConfigHook
quickgui> Setting up SDK
quickgui> Installing dependencies
quickgui> Finished dartConfigHook
quickgui> Running phase: buildPhase
quickgui> [  +14 ms] Failed obtaining PRETTY_NAME for Linux: PathNotFoundException: Cannot open file, path = '/usr/lib/os-release' (OS Error: No such file or directory, errno = 2)
quickgui> [  +16 ms] Unable to locate an Android SDK.
quickgui> [ +121 ms] Skipping pub get: generated by third-party.
quickgui> [  +59 ms] Found plugin url_launcher_web at /nix/store/02bwaa9svc88fyzbpffxic18475dzghl-pub-url_launcher_web-2.3.1/
quickgui> [  +43 ms] Found plugin window_size at /nix/store/281pj1qkmwsbzmn02r6q9icaj11x15ab-pub-window_size-0.1.0/plugins/window_size/
quickgui> [  +51 ms] Found plugin path_provider_linux at /nix/store/4k81c6xlrv9flldg38h9vz71yjynvbnn-pub-path_provider_linux-2.2.1/
quickgui> [  +28 ms] Found plugin shared_preferences_android at /nix/store/85nhyhivqhdfn8zcp59b4sxcz51nb4ja-pub-shared_preferences_android-2.2.3/
quickgui> [  +26 ms] Found plugin shared_preferences_foundation at /nix/store/aklaxr0m3qnidlyly2srv8jx4n28b3iz-pub-shared_preferences_foundation-2.4.0/
quickgui> [  +24 ms] Found plugin url_launcher_ios at /nix/store/c0y1cs8qg8iimamzm0y6g0nn7bz6l8cj-pub-url_launcher_ios-6.3.0/
quickgui> [  +14 ms] Found plugin flutter_plugin_android_lifecycle at /nix/store/fmpddwahgxhap1444frgg4vkd1nm0ic0-pub-flutter_plugin_android_lifecycle-2.0.20/
quickgui> [  +21 ms] Found plugin shared_preferences at /nix/store/iawy6vrk3vdj2q32pnqgf6amhp0dk3xh-pub-shared_preferences-2.2.3/
quickgui> [  +10 ms] Found plugin url_launcher_windows at /nix/store/ivk6fyy4sc5rm6xwq3rr2q3xicr4p5in-pub-url_launcher_windows-3.1.1/
quickgui> [  +28 ms] Found plugin url_launcher_android at /nix/store/ly3f4fiaspa5n4nak4sh23gxm6wdxr68-pub-url_launcher_android-6.3.3/
quickgui> [  +17 ms] Found plugin file_picker at /nix/store/nbksgwm99d12hkss1h768rx16jf7gx8h-pub-file_picker-8.0.6/
quickgui> [   +3 ms] Found plugin url_launcher_macos at /nix/store/nlrr9r5bzb9b1c82lhbx1n26db13ll5k-pub-url_launcher_macos-3.2.0/
quickgui> [   +6 ms] Found plugin shared_preferences_linux at /nix/store/pa9rc1ak6rpfbw7rnb3hn19385l0m6k5-pub-shared_preferences_linux-2.3.2/
quickgui> [   +6 ms] Found plugin shared_preferences_windows at /nix/store/qy21qcd19p1wwp1lb1nyj96sy81ag8js-pub-shared_preferences_windows-2.3.2/
quickgui> [  +12 ms] Found plugin path_provider_windows at /nix/store/wib0w6wj5clkiwglwfiddyiby1ssj6wk-pub-path_provider_windows-2.2.1/
quickgui> [   +3 ms] Found plugin url_launcher_linux at /nix/store/wrclapn89ya4jlcg654l0qwh6xlbw4l5-pub-url_launcher_linux-3.1.1/
quickgui> [   +7 ms] Found plugin package_info_plus at /nix/store/xqjqv966371lq9djmlrgmwkrsvf8wjsh-pub-package_info_plus-8.0.0/
quickgui> [   +6 ms] Found plugin url_launcher at /nix/store/yy83yg70as8yijm6vz1n633vv4fkjcaq-pub-url_launcher-6.3.0/
quickgui> [  +10 ms] Found plugin shared_preferences_web at /nix/store/zdjfqds2j65c0k1yqjx3dmy6xiqlci9h-pub-shared_preferences_web-2.3.0/
quickgui> [  +49 ms] Found plugin url_launcher_web at /nix/store/02bwaa9svc88fyzbpffxic18475dzghl-pub-url_launcher_web-2.3.1/
quickgui> [  +11 ms] Found plugin window_size at /nix/store/281pj1qkmwsbzmn02r6q9icaj11x15ab-pub-window_size-0.1.0/plugins/window_size/
quickgui> [  +14 ms] Found plugin path_provider_linux at /nix/store/4k81c6xlrv9flldg38h9vz71yjynvbnn-pub-path_provider_linux-2.2.1/
quickgui> [   +7 ms] Found plugin shared_preferences_android at /nix/store/85nhyhivqhdfn8zcp59b4sxcz51nb4ja-pub-shared_preferences_android-2.2.3/
quickgui> [   +7 ms] Found plugin shared_preferences_foundation at /nix/store/aklaxr0m3qnidlyly2srv8jx4n28b3iz-pub-shared_preferences_foundation-2.4.0/
quickgui> [   +6 ms] Found plugin url_launcher_ios at /nix/store/c0y1cs8qg8iimamzm0y6g0nn7bz6l8cj-pub-url_launcher_ios-6.3.0/
quickgui> [  +10 ms] Found plugin flutter_plugin_android_lifecycle at /nix/store/fmpddwahgxhap1444frgg4vkd1nm0ic0-pub-flutter_plugin_android_lifecycle-2.0.20/
quickgui> [   +6 ms] Found plugin shared_preferences at /nix/store/iawy6vrk3vdj2q32pnqgf6amhp0dk3xh-pub-shared_preferences-2.2.3/
quickgui> [   +1 ms] Found plugin url_launcher_windows at /nix/store/ivk6fyy4sc5rm6xwq3rr2q3xicr4p5in-pub-url_launcher_windows-3.1.1/
quickgui> [   +5 ms] Found plugin url_launcher_android at /nix/store/ly3f4fiaspa5n4nak4sh23gxm6wdxr68-pub-url_launcher_android-6.3.3/
quickgui> [   +4 ms] Found plugin file_picker at /nix/store/nbksgwm99d12hkss1h768rx16jf7gx8h-pub-file_picker-8.0.6/
quickgui> [   +1 ms] Found plugin url_launcher_macos at /nix/store/nlrr9r5bzb9b1c82lhbx1n26db13ll5k-pub-url_launcher_macos-3.2.0/
quickgui> [   +2 ms] Found plugin shared_preferences_linux at /nix/store/pa9rc1ak6rpfbw7rnb3hn19385l0m6k5-pub-shared_preferences_linux-2.3.2/
quickgui> [   +4 ms] Found plugin shared_preferences_windows at /nix/store/qy21qcd19p1wwp1lb1nyj96sy81ag8js-pub-shared_preferences_windows-2.3.2/
quickgui> [   +6 ms] Found plugin path_provider_windows at /nix/store/wib0w6wj5clkiwglwfiddyiby1ssj6wk-pub-path_provider_windows-2.2.1/
quickgui> [   +2 ms] Found plugin url_launcher_linux at /nix/store/wrclapn89ya4jlcg654l0qwh6xlbw4l5-pub-url_launcher_linux-3.1.1/
quickgui> [   +1 ms] Found plugin package_info_plus at /nix/store/xqjqv966371lq9djmlrgmwkrsvf8wjsh-pub-package_info_plus-8.0.0/
quickgui> [   +2 ms] Found plugin url_launcher at /nix/store/yy83yg70as8yijm6vz1n633vv4fkjcaq-pub-url_launcher-6.3.0/
quickgui> [   +3 ms] Found plugin shared_preferences_web at /nix/store/zdjfqds2j65c0k1yqjx3dmy6xiqlci9h-pub-shared_preferences_web-2.3.0/
quickgui> [  +61 ms] Building Linux application...
quickgui> [   +5 ms] executing: [build/linux/x64/release/] cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DFLUTTER_TARGET_PLATFORM=linux-x64 /build/source/linux
quickgui> [ +464 ms] -- The CXX compiler identification is Clang 18.1.8
quickgui> [ +113 ms] -- Detecting CXX compiler ABI info
quickgui> [ +439 ms] -- Detecting CXX compiler ABI info - done
quickgui> [ +122 ms] -- Check for working CXX compiler: /nix/store/qajfjv5lnfys2ykm0ggkr4lnnkvgs10r-clang-wrapper-18.1.8/bin/clang++ - skipped
quickgui> [        ] -- Detecting CXX compile features
quickgui> [   +1 ms] -- Detecting CXX compile features - done
quickgui> [  +10 ms] -- Found PkgConfig: /nix/store/946chn5ja4yrvnnc5izwv792cc57xpy8-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
quickgui> [        ] -- Checking for module 'gtk+-3.0'
quickgui> [  +64 ms] --   Found gtk+-3.0, version 3.24.43
quickgui> [ +477 ms] -- Checking for module 'glib-2.0'
quickgui> [  +51 ms] --   Found glib-2.0, version 2.82.1
quickgui> [  +90 ms] -- Checking for module 'gio-2.0'
quickgui> [  +38 ms] --   Found gio-2.0, version 2.82.1
quickgui> [ +176 ms] -- Configuring done (2.0s)
quickgui> [  +58 ms] -- Generating done (0.1s)
quickgui> [        ] -- Build files have been written to: /build/source/build/linux/x64/release
quickgui> [  +10 ms] executing: ninja -C build/linux/x64/release install
quickgui> [  +26 ms] ninja: Entering directory `build/linux/x64/release'
quickgui> [+14445 ms] [1/10] Generating /build/source/linux/flutter/ephemeral/libflutter_linux_gtk.so, /build/source/linux/flutter/ephemeral/flutter_linux/fl_basic_message_channel.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_binary_codec.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_binary_messenger.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_dart_project.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_engine.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_json_message_codec.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_json_method_codec.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_message_codec.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_method_call.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_method_channel.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_method_codec.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_method_response.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_plugin_registrar.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_plugin_registry.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_standard_message_codec.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_standard_method_codec.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_string_codec.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_value.h, /build/source/linux/flutter/ephemeral/flutter_linux/fl_view.h, /build/source/linux/flutter/ephemeral/flutter_linux/flutter_linux.h, _phony_
quickgui> [   +8 ms] FAILED: /build/source/linux/flutter/ephemeral/libflutter_linux_gtk.so /build/source/linux/flutter/ephemeral/flutter_linux/fl_basic_message_channel.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_binary_codec.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_binary_messenger.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_dart_project.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_engine.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_json_message_codec.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_json_method_codec.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_message_codec.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_method_call.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_method_channel.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_method_codec.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_method_response.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_plugin_registrar.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_plugin_registry.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_standard_message_codec.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_standard_method_codec.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_string_codec.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_value.h /build/source/linux/flutter/ephemeral/flutter_linux/fl_view.h /build/source/linux/flutter/ephemeral/flutter_linux/flutter_linux.h flutter/_phony_ /build/source/build/linux/x64/release/flutter/_phony_ 
quickgui> [        ] cd /build/source/build/linux/x64/release/flutter && /nix/store/irwcgpm8csj7br49jih6kdpqkaqya3vf-cmake-3.30.4/bin/cmake -E env FLUTTER_ROOT=/nix/store/d1ppwlksa636sw80y0flxs2jdyq0yr07-flutter-wrapped-3.24.4-sdk-links PROJECT_DIR=/build/source DART_OBFUSCATION=false SPLIT_DEBUG_INFO=/nix/store/nxxw289lb0j8pb7ax3p9mmx5swzzqp6q-quickgui-1.2.10-debug TRACK_WIDGET_CREATION=true TREE_SHAKE_ICONS=true PACKAGE_CONFIG=/build/source/.dart_tool/package_config.json FLUTTER_TARGET=lib/main.dart /nix/store/d1ppwlksa636sw80y0flxs2jdyq0yr07-flutter-wrapped-3.24.4-sdk-links/packages/flutter_tools/bin/tool_backend.sh linux-x64 Release
quickgui> [        ] [  +10 ms] Failed obtaining PRETTY_NAME for Linux: PathNotFoundException: Cannot open file, path = '/usr/lib/os-release' (OS Error: No such file or directory, errno = 2)
quickgui> [   +1 ms] [  +17 ms] Unable to locate an Android SDK.
quickgui> [        ] [ +137 ms] Initializing file store
quickgui> [        ] [  +18 ms] Skipping target: gen_localizations
quickgui> [        ] [   +4 ms] invalidated build due to missing files: /build/source/.dart_tool/package_config_subset
quickgui> [        ] [   +1 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputMissing: The following inputs were missing: /build/source/.dart_tool/package_config_subset}
quickgui> [        ] [   +7 ms] unpack_linux: Starting due to {}
quickgui> [        ] [ +141 ms] unpack_linux: Complete
quickgui> [        ] [ +208 ms] Found plugin url_launcher_web at /nix/store/02bwaa9svc88fyzbpffxic18475dzghl-pub-url_launcher_web-2.3.1/
quickgui> [        ] [  +20 ms] Found plugin window_size at /nix/store/281pj1qkmwsbzmn02r6q9icaj11x15ab-pub-window_size-0.1.0/plugins/window_size/
quickgui> [        ] [  +22 ms] Found plugin path_provider_linux at /nix/store/4k81c6xlrv9flldg38h9vz71yjynvbnn-pub-path_provider_linux-2.2.1/
quickgui> [        ] [   +9 ms] Found plugin shared_preferences_android at /nix/store/85nhyhivqhdfn8zcp59b4sxcz51nb4ja-pub-shared_preferences_android-2.2.3/
quickgui> [        ] [  +10 ms] Found plugin shared_preferences_foundation at /nix/store/aklaxr0m3qnidlyly2srv8jx4n28b3iz-pub-shared_preferences_foundation-2.4.0/
quickgui> [        ] [  +11 ms] Found plugin url_launcher_ios at /nix/store/c0y1cs8qg8iimamzm0y6g0nn7bz6l8cj-pub-url_launcher_ios-6.3.0/
quickgui> [        ] [  +11 ms] Found plugin flutter_plugin_android_lifecycle at /nix/store/fmpddwahgxhap1444frgg4vkd1nm0ic0-pub-flutter_plugin_android_lifecycle-2.0.20/
quickgui> [        ] [  +10 ms] Found plugin shared_preferences at /nix/store/iawy6vrk3vdj2q32pnqgf6amhp0dk3xh-pub-shared_preferences-2.2.3/
quickgui> [        ] [   +2 ms] Found plugin url_launcher_windows at /nix/store/ivk6fyy4sc5rm6xwq3rr2q3xicr4p5in-pub-url_launcher_windows-3.1.1/
quickgui> [        ] [  +12 ms] Found plugin url_launcher_android at /nix/store/ly3f4fiaspa5n4nak4sh23gxm6wdxr68-pub-url_launcher_android-6.3.3/
quickgui> [        ] [  +12 ms] Found plugin file_picker at /nix/store/nbksgwm99d12hkss1h768rx16jf7gx8h-pub-file_picker-8.0.6/
quickgui> [        ] [   +2 ms] Found plugin url_launcher_macos at /nix/store/nlrr9r5bzb9b1c82lhbx1n26db13ll5k-pub-url_launcher_macos-3.2.0/
quickgui> [        ] [   +3 ms] Found plugin shared_preferences_linux at /nix/store/pa9rc1ak6rpfbw7rnb3hn19385l0m6k5-pub-shared_preferences_linux-2.3.2/
quickgui> [        ] [   +2 ms] Found plugin shared_preferences_windows at /nix/store/qy21qcd19p1wwp1lb1nyj96sy81ag8js-pub-shared_preferences_windows-2.3.2/
quickgui> [        ] [   +7 ms] Found plugin path_provider_windows at /nix/store/wib0w6wj5clkiwglwfiddyiby1ssj6wk-pub-path_provider_windows-2.2.1/
quickgui> [        ] [   +2 ms] Found plugin url_launcher_linux at /nix/store/wrclapn89ya4jlcg654l0qwh6xlbw4l5-pub-url_launcher_linux-3.1.1/
quickgui> [        ] [   +3 ms] Found plugin package_info_plus at /nix/store/xqjqv966371lq9djmlrgmwkrsvf8wjsh-pub-package_info_plus-8.0.0/
quickgui> [        ] [   +5 ms] Found plugin url_launcher at /nix/store/yy83yg70as8yijm6vz1n633vv4fkjcaq-pub-url_launcher-6.3.0/
quickgui> [        ] [   +5 ms] Found plugin shared_preferences_web at /nix/store/zdjfqds2j65c0k1yqjx3dmy6xiqlci9h-pub-shared_preferences_web-2.3.0/
quickgui> [        ] [  +31 ms] gen_dart_plugin_registrant: Complete
quickgui> [        ] [        ] kernel_snapshot_program: Starting due to {}
quickgui> [   +1 ms] [  +10 ms] /nix/store/d1ppwlksa636sw80y0flxs2jdyq0yr07-flutter-wrapped-3.24.4-sdk-links/bin/cache/dart-sdk/bin/dartaotruntime --disable-dart-dev /nix/store/d1ppwlksa636sw80y0flxs2jdyq0yr07-flutter-wrapped-3.24.4-sdk-links/bin/cache/dart-sdk/bin/snapshots/frontend_server_aot.dart.snapshot --sdk-root /nix/store/d1ppwlksa636sw80y0flxs2jdyq0yr07-flutter-wrapped-3.24.4-sdk-links/bin/cache/artifacts/engine/common/flutter_patched_sdk_product/ --target=flutter --no-print-incremental-dependencies -Ddart.vm.profile=false -Ddart.vm.product=true --delete-tostring-package-uri=dart:ui --delete-tostring-package-uri=package:flutter --aot --tfa --target-os linux --packages /build/source/.dart_tool/package_config.json --output-dill /build/source/.dart_tool/flutter_build/039decc5a3fdc4dcd63b4de0ed5383a1/program.dill --depfile /build/source/.dart_tool/flutter_build/039decc5a3fdc4dcd63b4de0ed5383a1/kernel_snapshot_program.d --source file:///build/source/.dart_tool/flutter_build/dart_plugin_registrant.dart --source package:flutter/src/dart_plugin_registrant.dart -Dflutter.dart_plugin_registrant=file:///build/source/.dart_tool/flutter_build/dart_plugin_registrant.dart --verbosity=error package:quickgui/main.dart
quickgui> [        ] [+4813 ms] /nix/store/bgvqb8pa4vspr3abxxyww7fdxqdqwmjr-flutter-sdk-flutter/lib/src/material/color_scheme.dart:1915:39: Error: Method not found: 'SchemeRainbow'.
quickgui> [        ] [   +4 ms]       DynamicSchemeVariant.rainbow => SchemeRainbow(sourceColorHct: sourceColor, isDark: isDark, contrastLevel: contrastLevel),
quickgui> [        ] [        ]                                       ^^^^^^^^^^^^^
quickgui> [        ] [        ] /nix/store/bgvqb8pa4vspr3abxxyww7fdxqdqwmjr-flutter-sdk-flutter/lib/src/material/color_scheme.dart:1916:42: Error: Method not found: 'SchemeFruitSalad'.
quickgui> [        ] [        ]       DynamicSchemeVariant.fruitSalad => SchemeFruitSalad(sourceColorHct: sourceColor, isDark: isDark, contrastLevel: contrastLevel),
quickgui> [        ] [        ]                                          ^^^^^^^^^^^^^^^^
quickgui> [        ] [+8347 ms] Persisting file store
quickgui> [        ] [   +5 ms] Done persisting file store
quickgui> [        ] [   +3 ms] Target kernel_snapshot_program failed: Exception
quickgui> [        ]            #0      KernelSnapshotProgram.build (package:flutter_tools/src/build_system/targets/common.dart:276:7)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #1      _BuildInstance._invokeInternal (package:flutter_tools/src/build_system/build_system.dart:875:9)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #2      Future.wait.<anonymous closure> (dart:async/future.dart:534:21)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #3      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:813:32)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #4      Future.wait.<anonymous closure> (dart:async/future.dart:534:21)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #5      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:813:32)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #6      Future.wait.<anonymous closure> (dart:async/future.dart:534:21)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #7      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:813:32)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #8      Future.wait.<anonymous closure> (dart:async/future.dart:534:21)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #9      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:813:32)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #10     Future.wait.<anonymous closure> (dart:async/future.dart:534:21)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #11     _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:813:32)
quickgui> [        ]            <asynchronous suspension>
quickgui> [   +2 ms]            #12     Future.wait.<anonymous closure> (dart:async/future.dart:534:21)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #13     _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:813:32)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #14     FlutterBuildSystem.build (package:flutter_tools/src/build_system/build_system.dart:635:16)
quickgui> [        ]            <asynchronous suspension>
quickgui> [   +3 ms]            #15     AssembleCommand.runCommand (package:flutter_tools/src/commands/assemble.dart:328:32)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #16     FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1408:27)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #17     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #18     CommandRunner.runCommand (package:args/command_runner.dart:212:13)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #19     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:416:9)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #20     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #21     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:364:5)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #22     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:130:9)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #23     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #24     main (package:flutter_tools/executable.dart:93:3)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ] [   +8 ms] "flutter assemble" took 13,840ms.
quickgui> [        ] [   +3 ms] 
quickgui> [        ]            #0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
quickgui> [        ]            #1      AssembleCommand.runCommand (package:flutter_tools/src/commands/assemble.dart:345:7)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #2      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1408:27)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #3      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #4      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #5      FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:416:9)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #6      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #7      FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:364:5)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #8      run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:130:9)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #9      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ]            #10     main (package:flutter_tools/executable.dart:93:3)
quickgui> [        ]            <asynchronous suspension>
quickgui> [        ] [        ] Running 1 shutdown hook
quickgui> [        ] [        ] Shutdown hooks complete
quickgui> [        ] [   +2 ms] exiting with code 1
quickgui> [        ] ninja: build stopped: subcommand failed.
quickgui> [  +17 ms] Building Linux application... (completed in 16.6s)
quickgui> [   +1 ms] "flutter linux" took 17,316ms.
quickgui> [   +3 ms] Build process failed
quickgui> [        ] 
quickgui>            #0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
quickgui>            #1      _runBuild (package:flutter_tools/src/linux/build_linux.dart:210:5)
quickgui>            <asynchronous suspension>
quickgui>            #2      buildLinux (package:flutter_tools/src/linux/build_linux.dart:83:5)
quickgui>            <asynchronous suspension>
quickgui>            #3      BuildLinuxCommand.runCommand (package:flutter_tools/src/commands/build_linux.dart:87:5)
quickgui>            <asynchronous suspension>
quickgui>            #4      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1408:27)
quickgui>            <asynchronous suspension>
quickgui>            #5      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
quickgui>            <asynchronous suspension>
quickgui>            #6      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
quickgui>            <asynchronous suspension>
quickgui>            #7      FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:416:9)
quickgui>            <asynchronous suspension>
quickgui>            #8      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
quickgui>            <asynchronous suspension>
quickgui>            #9      FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:364:5)
quickgui>            <asynchronous suspension>
quickgui>            #10     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:130:9)
quickgui>            <asynchronous suspension>
quickgui>            #11     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
quickgui>            <asynchronous suspension>
quickgui>            #12     main (package:flutter_tools/executable.dart:93:3)
quickgui>            <asynchronous suspension>

...

```

--- a/flake.nix
+++ b/flake.nix
@@ -16,7 +16,7 @@
       # Define supported systems and a helper function for generating system-specific outputs
       #TODO: Add the following as quickemu/quickgui/GitHub builders support them:
       #      aarch64-darwin aarch64-linux x86_64-darwin
-      supportedSystems = [ "x86_64-linux" ];
+      supportedSystems = [ "aarch64-linux" "x86_64-linux" ];
 
       forEachSupportedSystem = f: nixpkgs.lib.genAttrs supportedSystems (system: f {
         system = system;

--- a/pubspec.lock.json
+++ b/pubspec.lock.json
@@ -4,21 +4,21 @@
       "dependency": "transitive",
       "description": {
         "name": "_discoveryapis_commons",
-        "sha256": "f8bb1fdbd77f3d5c1d62b5b0eca75fbf1e41bf4f6c62628f880582e2182ae45d",
+        "sha256": "113c4100b90a5b70a983541782431b82168b3cae166ab130649c36eb3559d498",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.0.6"
+      "version": "1.0.7"
     },
     "ansicolor": {
       "dependency": "transitive",
       "description": {
         "name": "ansicolor",
-        "sha256": "8bf17a8ff6ea17499e40a2d2542c2f481cd7615760c6d34065cb22bfd22e6880",
+        "sha256": "50e982d500bc863e1d703448afdbf9e5a72eb48840a4f766fa361ffd6877055f",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.0.2"
+      "version": "2.0.3"
     },
     "archive": {
       "dependency": "transitive",
@@ -34,11 +34,11 @@
       "dependency": "transitive",
       "description": {
         "name": "args",
-        "sha256": "7cf60b9f0cc88203c5a190b4cd62a99feea42759a7fa695010eb5de1c0b2252a",
+        "sha256": "bf9f5caeea8d8fe6721a9c358dd8a5c1947b27f1cfaa18b39c301273594919e6",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.5.0"
+      "version": "2.6.0"
     },
     "async": {
       "dependency": "transitive",
@@ -94,11 +94,11 @@
       "dependency": "transitive",
       "description": {
         "name": "cli_util",
-        "sha256": "c05b7406fdabc7a49a3929d4af76bcaccbbffcbcdcf185b082e1ae07da323d19",
+        "sha256": "ff6785f7e9e3c38ac98b2fb035701789de90154024a75b6cb926445e83197d1c",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "0.4.1"
+      "version": "0.4.2"
     },
     "clock": {
       "dependency": "transitive",
@@ -134,21 +134,21 @@
       "dependency": "transitive",
       "description": {
         "name": "cross_file",
-        "sha256": "55d7b444feb71301ef6b8838dbc1ae02e63dd48c8773f3810ff53bb1e2945b32",
+        "sha256": "7caf6a750a0c04effbb52a676dce9a4a592e10ad35c34d6d2d0e4811160d5670",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "0.3.4+1"
+      "version": "0.3.4+2"
     },
     "crypto": {
       "dependency": "transitive",
       "description": {
         "name": "crypto",
-        "sha256": "ff625774173754681d66daaf4a448684fb04b78f902da9cb3d308c19cc5e8bab",
+        "sha256": "1e445881f28f22d6140f181e07737b22f1e099a5e1ff94b0af2f9e4a463f4855",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.0.3"
+      "version": "3.0.6"
     },
     "cupertino_icons": {
       "dependency": "direct main",
@@ -184,11 +184,21 @@
       "dependency": "transitive",
       "description": {
         "name": "dio",
-        "sha256": "11e40df547d418cc0c4900a9318b26304e665da6fa4755399a9ff9efd09034b5",
+        "sha256": "5598aa796bbf4699afd5c67c0f5f6e2ed542afc956884b9cd58c306966efc260",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "5.4.3+1"
+      "version": "5.7.0"
+    },
+    "dio_web_adapter": {
+      "dependency": "transitive",
+      "description": {
+        "name": "dio_web_adapter",
+        "sha256": "33259a9276d6cea88774a0000cfae0d861003497755969c92faa223108620dc8",
+        "url": "https://pub.dev"
+      },
+      "source": "hosted",
+      "version": "2.0.0"
     },
     "fake_async": {
       "dependency": "transitive",
@@ -204,31 +214,31 @@
       "dependency": "transitive",
       "description": {
         "name": "ffi",
-        "sha256": "493f37e7df1804778ff3a53bd691d8692ddf69702cf4c1c1096a2e41b4779e21",
+        "sha256": "16ed7b077ef01ad6170a3d0c57caa4a112a38d7a2ed5602e0aca9ca6f3d98da6",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.1.2"
+      "version": "2.1.3"
     },
     "file": {
       "dependency": "transitive",
       "description": {
         "name": "file",
-        "sha256": "5fc22d7c25582e38ad9a8515372cd9a93834027aacf1801cf01164dac0ffa08c",
+        "sha256": "a3b4f84adafef897088c160faf7dfffb7696046cb13ae90b508c2cbc95d3b8d4",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "7.0.0"
+      "version": "7.0.1"
     },
     "file_picker": {
       "dependency": "direct main",
       "description": {
         "name": "file_picker",
-        "sha256": "824f5b9f389bfc4dddac3dea76cd70c51092d9dff0b2ece7ef4f53db8547d258",
+        "sha256": "16dc141db5a2ccc6520ebb6a2eb5945b1b09e95085c021d9f914f8ded7f1465c",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "8.0.6"
+      "version": "8.1.4"
     },
     "flutter": {
       "dependency": "direct main",
@@ -296,21 +306,21 @@
       "dependency": "transitive",
       "description": {
         "name": "flutter_plugin_android_lifecycle",
-        "sha256": "c6b0b4c05c458e1c01ad9bcc14041dd7b1f6783d487be4386f793f47a8a4d03e",
+        "sha256": "9b78450b89f059e96c9ebb355fa6b3df1d6b330436e0b885fb49594c41721398",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.0.20"
+      "version": "2.0.23"
     },
     "flutter_svg": {
       "dependency": "direct main",
       "description": {
         "name": "flutter_svg",
-        "sha256": "7b4ca6cf3304575fe9c8ec64813c8d02ee41d2afe60bcfe0678bcb5375d596a2",
+        "sha256": "54900a1a1243f3c4a5506d853a2b5c2dbc38d5f27e52a52618a8054401431123",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.0.10+1"
+      "version": "2.0.16"
     },
     "flutter_test": {
       "dependency": "direct dev",
@@ -378,11 +388,11 @@
       "dependency": "transitive",
       "description": {
         "name": "google_identity_services_web",
-        "sha256": "9482364c9f8b7bd36902572ebc3a7c2b5c8ee57a9c93e6eb5099c1a9ec5265d8",
+        "sha256": "55580f436822d64c8ff9a77e37d61f5fb1e6c7ec9d632a43ee324e2a05c3c6c9",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "0.3.1+1"
+      "version": "0.3.3"
     },
     "googleapis": {
       "dependency": "transitive",
@@ -408,11 +418,11 @@
       "dependency": "transitive",
       "description": {
         "name": "http",
-        "sha256": "761a297c042deedc1ffbb156d6e2af13886bb305c2a343a4d972504cd67dd938",
+        "sha256": "b9c29a161230ee03d3ccf545097fccd9b87a5264228c5d348202e0f0c28f9010",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.2.1"
+      "version": "1.2.2"
     },
     "http_parser": {
       "dependency": "transitive",
@@ -428,11 +438,11 @@
       "dependency": "transitive",
       "description": {
         "name": "image",
-        "sha256": "2237616a36c0d69aef7549ab439b833fb7f9fb9fc861af2cc9ac3eedddd69ca8",
+        "sha256": "f31d52537dc417fdcde36088fdf11d191026fd5e4fae742491ebd40e5a8bea7d",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "4.2.0"
+      "version": "4.3.0"
     },
     "intl": {
       "dependency": "transitive",
@@ -468,21 +478,21 @@
       "dependency": "transitive",
       "description": {
         "name": "leak_tracker",
-        "sha256": "7f0df31977cb2c0b88585095d168e689669a2cc9b97c309665e3386f3e9d341a",
+        "sha256": "3f87a60e8c63aecc975dda1ceedbc8f24de75f09e4856ea27daf8958f2f0ce05",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "10.0.4"
+      "version": "10.0.5"
     },
     "leak_tracker_flutter_testing": {
       "dependency": "transitive",
       "description": {
         "name": "leak_tracker_flutter_testing",
-        "sha256": "06e98f569d004c1315b991ded39924b21af84cf14cc94791b8aea337d25b57f8",
+        "sha256": "932549fb305594d82d7183ecd9fa93463e9914e1b67cacc34bc40906594a1806",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.0.3"
+      "version": "3.0.5"
     },
     "leak_tracker_testing": {
       "dependency": "transitive",
@@ -518,11 +528,11 @@
       "dependency": "transitive",
       "description": {
         "name": "logging",
-        "sha256": "623a88c9594aa774443aa3eb2d41807a48486b5613e67599fb4c41c0ad47c340",
+        "sha256": "c8245ada5f1717ed44271ed1c26b8ce85ca3228fd2ffdb75468ab01979309d61",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.2.0"
+      "version": "1.3.0"
     },
     "matcher": {
       "dependency": "transitive",
@@ -538,31 +548,31 @@
       "dependency": "transitive",
       "description": {
         "name": "material_color_utilities",
-        "sha256": "0e0a020085b65b6083975e499759762399b4475f766c21668c4ecca34ea74e5a",
+        "sha256": "f7142bb1154231d7ea5f96bc7bde4bda2a0945d2806bb11670e30b850d56bdec",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "0.8.0"
+      "version": "0.11.1"
     },
     "meta": {
       "dependency": "transitive",
       "description": {
         "name": "meta",
-        "sha256": "7687075e408b093f36e6bbf6c91878cc0d4cd10f409506f7bc996f68220b9136",
+        "sha256": "bdb68674043280c3428e9ec998512fb681678676b3c54e773629ffe74419f8c7",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.12.0"
+      "version": "1.15.0"
     },
     "msix": {
       "dependency": "transitive",
       "description": {
         "name": "msix",
-        "sha256": "519b183d15dc9f9c594f247e2d2339d855cf0eaacc30e19b128e14f3ecc62047",
+        "sha256": "c50d6bd1aafe0d071a3c1e5a5ccb056404502935cb0a549e3178c4aae16caf33",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.16.7"
+      "version": "3.16.8"
     },
     "mustache_template": {
       "dependency": "transitive",
@@ -598,21 +608,21 @@
       "dependency": "direct main",
       "description": {
         "name": "package_info_plus",
-        "sha256": "b93d8b4d624b4ea19b0a5a208b2d6eff06004bc3ce74c06040b120eeadd00ce0",
+        "sha256": "da8d9ac8c4b1df253d1a328b7bf01ae77ef132833479ab40763334db13b91cce",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "8.0.0"
+      "version": "8.1.1"
     },
     "package_info_plus_platform_interface": {
       "dependency": "transitive",
       "description": {
         "name": "package_info_plus_platform_interface",
-        "sha256": "f49918f3433a3146047372f9d4f1f847511f2acd5cd030e1f44fe5a50036b70e",
+        "sha256": "ac1f4a4847f1ade8e6a87d1f39f5d7c67490738642e2542f559ec38c37489a66",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.0.0"
+      "version": "3.0.1"
     },
     "parse_app_package": {
       "dependency": "transitive",
@@ -638,11 +648,11 @@
       "dependency": "transitive",
       "description": {
         "name": "path_parsing",
-        "sha256": "e3e67b1629e6f7e8100b367d3db6ba6af4b1f0bb80f64db18ef1fbabd2fa9ccf",
+        "sha256": "883402936929eac138ee0a45da5b0f2c80f89913e6dc3bf77eb65b84b409c6ca",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.0.1"
+      "version": "1.1.0"
     },
     "path_provider_linux": {
       "dependency": "transitive",
@@ -668,11 +678,11 @@
       "dependency": "transitive",
       "description": {
         "name": "path_provider_windows",
-        "sha256": "8bc9f22eee8690981c22aa7fc602f5c85b497a6fb2ceb35ee5a5e5ed85ad8170",
+        "sha256": "bd6f00dbd873bfb70d0761682da2b3a2c2fccc2b9e84c495821639601d81afe7",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.2.1"
+      "version": "2.3.0"
     },
     "petitparser": {
       "dependency": "transitive",
@@ -688,11 +698,11 @@
       "dependency": "transitive",
       "description": {
         "name": "platform",
-        "sha256": "9b71283fc13df574056616011fb138fd3b793ea47cc509c189a6c3fa5f8a1a65",
+        "sha256": "5d6b1b0036a5f331ebc77c850ebc8506cbc1e9416c27e59b439f917a902a4984",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.1.5"
+      "version": "3.1.6"
     },
     "plist_parser": {
       "dependency": "transitive",
@@ -718,11 +728,11 @@
       "dependency": "direct main",
       "description": {
         "name": "process_run",
-        "sha256": "6052115540ad88715d6bcee60656970f70c68c85846d1948b92e435f0382899e",
+        "sha256": "a68fa9727392edad97a2a96a77ce8b0c17d28336ba1b284b1dfac9595a4299ea",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.0.0+1"
+      "version": "1.2.2+1"
     },
     "provider": {
       "dependency": "direct main",
@@ -768,11 +778,11 @@
       "dependency": "direct main",
       "description": {
         "name": "quiver",
-        "sha256": "b1c1ac5ce6688d77f65f3375a9abb9319b3cb32486bdc7a1e0fdf004d7ba4e47",
+        "sha256": "ea0b925899e64ecdfbf9c7becb60d5b50e706ade44a85b2363be2a22d88117d2",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.2.1"
+      "version": "3.2.2"
     },
     "recase": {
       "dependency": "transitive",
@@ -788,71 +798,71 @@
       "dependency": "direct main",
       "description": {
         "name": "shared_preferences",
-        "sha256": "d3bbe5553a986e83980916ded2f0b435ef2e1893dfaa29d5a7a790d0eca12180",
+        "sha256": "95f9997ca1fb9799d494d0cb2a780fd7be075818d59f00c43832ed112b158a82",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.2.3"
+      "version": "2.3.3"
     },
     "shared_preferences_android": {
       "dependency": "transitive",
       "description": {
         "name": "shared_preferences_android",
-        "sha256": "93d0ec9dd902d85f326068e6a899487d1f65ffcd5798721a95330b26c8131577",
+        "sha256": "7f172d1b06de5da47b6264c2692ee2ead20bbbc246690427cdb4fc301cd0c549",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.2.3"
+      "version": "2.3.4"
     },
     "shared_preferences_foundation": {
       "dependency": "transitive",
       "description": {
         "name": "shared_preferences_foundation",
-        "sha256": "0a8a893bf4fd1152f93fec03a415d11c27c74454d96e2318a7ac38dd18683ab7",
+        "sha256": "07e050c7cd39bad516f8d64c455f04508d09df104be326d8c02551590a0d513d",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.4.0"
+      "version": "2.5.3"
     },
     "shared_preferences_linux": {
       "dependency": "transitive",
       "description": {
         "name": "shared_preferences_linux",
-        "sha256": "9f2cbcf46d4270ea8be39fa156d86379077c8a5228d9dfdb1164ae0bb93f1faa",
+        "sha256": "580abfd40f415611503cae30adf626e6656dfb2f0cee8f465ece7b6defb40f2f",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.3.2"
+      "version": "2.4.1"
     },
     "shared_preferences_platform_interface": {
       "dependency": "transitive",
       "description": {
         "name": "shared_preferences_platform_interface",
-        "sha256": "22e2ecac9419b4246d7c22bfbbda589e3acf5c0351137d87dd2939d984d37c3b",
+        "sha256": "57cbf196c486bc2cf1f02b85784932c6094376284b3ad5779d1b1c6c6a816b80",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.3.2"
+      "version": "2.4.1"
     },
     "shared_preferences_web": {
       "dependency": "transitive",
       "description": {
         "name": "shared_preferences_web",
-        "sha256": "9aee1089b36bd2aafe06582b7d7817fd317ef05fc30e6ba14bff247d0933042a",
+        "sha256": "d2ca4132d3946fec2184261726b355836a82c33d7d5b67af32692aff18a4684e",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.3.0"
+      "version": "2.4.2"
     },
     "shared_preferences_windows": {
       "dependency": "transitive",
       "description": {
         "name": "shared_preferences_windows",
-        "sha256": "841ad54f3c8381c480d0c9b508b89a34036f512482c407e6df7a9c4aa2ef8f59",
+        "sha256": "94ef0f72b2d71bc3e700e025db3710911bd51a71cefb65cc609dd0d9a982e3c1",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.3.2"
+      "version": "2.4.1"
     },
     "shell_executor": {
       "dependency": "transitive",
@@ -924,11 +934,11 @@
       "dependency": "transitive",
       "description": {
         "name": "synchronized",
-        "sha256": "539ef412b170d65ecdafd780f924e5be3f60032a1128df156adad6c5b373d558",
+        "sha256": "69fe30f3a8b04a0be0c15ae6490fc859a78ef4c43ae2dd5e8a623d45bfcf9225",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.1.0+1"
+      "version": "3.3.0+3"
     },
     "term_glyph": {
       "dependency": "transitive",
@@ -944,11 +954,11 @@
       "dependency": "transitive",
       "description": {
         "name": "test_api",
-        "sha256": "9955ae474176f7ac8ee4e989dadfb411a58c30415bcfb648fa04b2b8a03afa7f",
+        "sha256": "5b8a98dafc4d5c4c9c72d8b31ab2b23fc13422348d2997120294d3bac86b4ddb",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "0.7.0"
+      "version": "0.7.2"
     },
     "tuple": {
       "dependency": "direct main",
@@ -964,61 +974,61 @@
       "dependency": "transitive",
       "description": {
         "name": "typed_data",
-        "sha256": "facc8d6582f16042dd49f2463ff1bd6e2c9ef9f3d5da3d9b087e244a7b564b3c",
+        "sha256": "f9049c039ebfeb4cf7a7104a675823cd72dba8297f264b6637062516699fa006",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.3.2"
+      "version": "1.4.0"
     },
     "url_launcher": {
       "dependency": "direct main",
       "description": {
         "name": "url_launcher",
-        "sha256": "21b704ce5fa560ea9f3b525b43601c678728ba46725bab9b01187b4831377ed3",
+        "sha256": "9d06212b1362abc2f0f0d78e6f09f726608c74e3b9462e8368bb03314aa8d603",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "6.3.0"
+      "version": "6.3.1"
     },
     "url_launcher_android": {
       "dependency": "transitive",
       "description": {
         "name": "url_launcher_android",
-        "sha256": "ceb2625f0c24ade6ef6778d1de0b2e44f2db71fded235eb52295247feba8c5cf",
+        "sha256": "6fc2f56536ee873eeb867ad176ae15f304ccccc357848b351f6f0d8d4a40d193",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "6.3.3"
+      "version": "6.3.14"
     },
     "url_launcher_ios": {
       "dependency": "transitive",
       "description": {
         "name": "url_launcher_ios",
-        "sha256": "7068716403343f6ba4969b4173cbf3b84fc768042124bc2c011e5d782b24fe89",
+        "sha256": "e43b677296fadce447e987a2f519dcf5f6d1e527dc35d01ffab4fff5b8a7063e",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "6.3.0"
+      "version": "6.3.1"
     },
     "url_launcher_linux": {
       "dependency": "transitive",
       "description": {
         "name": "url_launcher_linux",
-        "sha256": "ab360eb661f8879369acac07b6bb3ff09d9471155357da8443fd5d3cf7363811",
+        "sha256": "4e9ba368772369e3e08f231d2301b4ef72b9ff87c31192ef471b380ef29a4935",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.1.1"
+      "version": "3.2.1"
     },
     "url_launcher_macos": {
       "dependency": "transitive",
       "description": {
         "name": "url_launcher_macos",
-        "sha256": "9a1a42d5d2d95400c795b2914c36fdcb525870c752569438e4ebb09a2b5d90de",
+        "sha256": "769549c999acdb42b8bcfa7c43d72bf79a382ca7441ab18a808e101149daf672",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.2.0"
+      "version": "3.2.1"
     },
     "url_launcher_platform_interface": {
       "dependency": "transitive",
@@ -1034,21 +1044,21 @@
       "dependency": "transitive",
       "description": {
         "name": "url_launcher_web",
-        "sha256": "8d9e750d8c9338601e709cd0885f95825086bd8b642547f26bda435aade95d8a",
+        "sha256": "772638d3b34c779ede05ba3d38af34657a05ac55b06279ea6edd409e323dca8e",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.3.1"
+      "version": "2.3.3"
     },
     "url_launcher_windows": {
       "dependency": "transitive",
       "description": {
         "name": "url_launcher_windows",
-        "sha256": "ecf9725510600aa2bb6d7ddabe16357691b6d2805f66216a97d1b881e21beff7",
+        "sha256": "44cf3aabcedde30f2dba119a9dea3b0f2672fbe6fa96e85536251d678216b3c4",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.1.1"
+      "version": "3.1.3"
     },
     "uuid": {
       "dependency": "transitive",
@@ -1064,31 +1074,31 @@
       "dependency": "transitive",
       "description": {
         "name": "vector_graphics",
-        "sha256": "32c3c684e02f9bc0afb0ae0aa653337a2fe022e8ab064bcd7ffda27a74e288e3",
+        "sha256": "27d5fefe86fb9aace4a9f8375b56b3c292b64d8c04510df230f849850d912cb7",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.1.11+1"
+      "version": "1.1.15"
     },
     "vector_graphics_codec": {
       "dependency": "transitive",
       "description": {
         "name": "vector_graphics_codec",
-        "sha256": "c86987475f162fadff579e7320c7ddda04cd2fdeffbe1129227a85d9ac9e03da",
+        "sha256": "2430b973a4ca3c4dbc9999b62b8c719a160100dcbae5c819bae0cacce32c9cdb",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.1.11+1"
+      "version": "1.1.12"
     },
     "vector_graphics_compiler": {
       "dependency": "transitive",
       "description": {
         "name": "vector_graphics_compiler",
-        "sha256": "12faff3f73b1741a36ca7e31b292ddeb629af819ca9efe9953b70bd63fc8cd81",
+        "sha256": "1b4b9e706a10294258727674a340ae0d6e64a7231980f9f9a3d12e4b42407aad",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.1.11+1"
+      "version": "1.1.16"
     },
     "vector_math": {
       "dependency": "transitive",
@@ -1114,31 +1124,31 @@
       "dependency": "transitive",
       "description": {
         "name": "vm_service",
-        "sha256": "3923c89304b715fb1eb6423f017651664a03bf5f4b29983627c4da791f74a4ec",
+        "sha256": "5c5f338a667b4c644744b661f309fb8080bb94b18a7e91ef1dbd343bed00ed6d",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "14.2.1"
+      "version": "14.2.5"
     },
     "web": {
       "dependency": "transitive",
       "description": {
         "name": "web",
-        "sha256": "97da13628db363c635202ad97068d47c5b8aa555808e7a9411963c533b449b27",
+        "sha256": "cd3543bd5798f6ad290ea73d210f423502e71900302dde696f8bff84bf89a1cb",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "0.5.1"
+      "version": "1.1.0"
     },
     "win32": {
       "dependency": "transitive",
       "description": {
         "name": "win32",
-        "sha256": "a79dbe579cb51ecd6d30b17e0cae4e0ea15e2c0e66f69ad4198f22a6789e94f4",
+        "sha256": "8b338d4486ab3fbc0ba0db9f9b4f5239b6697fcee427939a40e720cbb9ee0a69",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "5.5.1"
+      "version": "5.9.0"
     },
     "window_size": {
       "dependency": "direct main",
@@ -1155,11 +1165,11 @@
       "dependency": "transitive",
       "description": {
         "name": "xdg_directories",
-        "sha256": "faea9dee56b520b55a566385b84f2e8de55e7496104adada9962e0bd11bcff1d",
+        "sha256": "7a3f37b05d989967cdddcbb571f1ea834867ae2faa29725fd085180e0883aa15",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.0.4"
+      "version": "1.1.0"
     },
     "xml": {
       "dependency": "transitive",
@@ -1183,7 +1193,7 @@
     }
   },
   "sdks": {
-    "dart": ">=3.4.0 <4.0.0",
-    "flutter": ">=3.22.0"
+    "dart": ">=3.5.0 <4.0.0",
+    "flutter": ">=3.24.0"
   }
 }

